cli/tests/permission_test.ts

```diff
@@ -16,13 +16,10 @@ const test: { [key: string]: (...args: any[]) => void | Promise<void> } = {
     Deno.listen({ transport: "tcp", port: 4541 });
   },
   runRequired(): void {
-    Deno.run({
-      cmd: [
-        "python",
-        "-c",
-        "import sys; sys.stdout.write('hello'); sys.stdout.flush()",
-      ],
+    const p = Deno.run({
+      cmd: ["printf", "hello"],
```

Comment:
@iammack `printf` is probably not available on Windows.


`echo` exists in DOS so that may be an x-platform way to write this.

Disclosure: I'm not a Windows user.